cluster-autoscaler/cloudprovider/gce/cache.go

```diff
@@ -305,6 +307,46 @@ func (gc *GceCache) InvalidateAllMigTargetSizes() {
 	gc.migTargetSizeCache = map[GceRef]int64{}
 }
 
+// GetMigInstanceTemplate returns the cached gce.InstanceTemplate for a mig GceRef
+func (gc *GceCache) GetMigInstanceTemplate(ref GceRef) (*gce.InstanceTemplate, bool) {
```

Comment:
Do you need to return found? It is hard to tell what it means without looking at implementation.
If you add nil to cache SetMigInstanceTemplate(some_ref, nil) then GetMigInstanceTemplate(some_ref) will return (nil, true). I think it is better to return nil in this case.

Reply:
I would leave it. It is consistent with how go generally deals with getting values from map. That there is separate bool value returned to determine if value was found or not.